components/git/git-node-metadata

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env node
+'use strict';
+
+const argv = require('../../lib/args')();
+const getMetadata = require('../metadata');
+const CLI = require('../../lib/cli');
+
+const logStream = process.stdout.isTTY ? process.stdout : process.stderr;
+const cli = new CLI(logStream);
+
+getMetadata(argv, cli).catch((err) => {
```

Comment:
Can you extend `argv` with configurations in `ncu/config`? Those are available via `require('../../lib/config').getMergedConfig()`.